Resources/Prototypes/Entities/Objects/Devices/pda.yml

```diff
@@ -116,6 +116,8 @@
       state: pda-internmed
   - type: Icon
     state: pda-internmed
+    - type: HealthAnalyzer
```

Comment:
You didn't indent this properly, follow 2 spaces for YAML.

Reply:
That SHOULD fix it, hopefully.